src/components/countries/country.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { BiChevronRightCircle } from 'react-icons/bi';
+import countryName from './countries_name';
+import './scss/country.scss';
+
+const Country = ({
+  country, totalConfirmed, handleClick,
+}) => {
+  const navigate = useNavigate();
+  const HandleRoute = (route) => {
+    navigate(route);
+  };
+  const isoName = countryName[country];
+
+  const mapImg = isoName ? `https://raw.githubusercontent.com/djaiss/mapsicon/33ba28808f8d32b5bae0ffada9cadd07073852e1/all/${isoName.toLowerCase()}/vector.svg` : '/World_map.png';
+  return (
+    <>
+      <button
+        className="country"
+        type="button"
+        onClick={() => {
+          HandleRoute('/country');
+          handleClick(country);
+        }}
+      >
+        <BiChevronRightCircle size={20} className="info-icon" color="white" />
+        <img className="country-map" alt="map" src={mapImg} width="100" height="100" />
+        <div className="country-data">
+          <h1 className="country-name">{country.toUpperCase()}</h1>
+          <span className="total-confirmed">{Intl.NumberFormat('de-DE').format(totalConfirmed)}</span>
+        </div>
+      </button>
+    </>
+
+  );
+};
```

Comment:
- [ ] Good job making sure that It's possible to navigate from the home page to the details page by clicking the individual category item. However, there should also be a link to navigate from the `Details` page back to the `Home` page by clicking "Back" button. See the image below:

<img width="136" alt="Loom_kmOqgdkcbf" src="https://user-images.githubusercontent.com/81468675/153694124-8c0ba780-27c5-4f58-9d70-81e02c585359.png">